### ISSUE TITLE
pip install peewee fails with Anaconda on Win10

### ISSUE THREAD

**Issue by happylrd** [coleifer/peewee#1549]
`pip install peewee` fails with Anaconda on Windows 10.
- OS: Windows 10
- Python: Python 3.6.4 :: Anaconda, Inc.
- Cython: 0.27.3

### error
```
playhouse\_sqlite_ext.c(531): fatal error C1083: Cannot open include file: 'sqlite3.h': No such file or directory
    error: command 'D:\\MyIDE\\VS2015\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2
```

**But when I used official Python without Anaconda, it installed successfully**.So what can I do for installing peewee successfully with Python in Anaconda?

**Comment by coleifer** [comment]
Hmm... I don't know, with setup.py, how to check if a header file/shared lib is going to be available ahead-of-time.

**Comment by happylrd** [comment]
@coleifer  `conda install -c conda-forge peewee` can solve this problem, the only regrettable thing is that the version of peewee is a little old, just 2.10.2.

**Comment by coleifer** [comment]
Ew, yeah... Peewee has much more extensive integration with SQLite in 3.x, although it's restricted to a module in `playhouse`. I'm not sure how to conditionally compile that, if Cython and a C compiler is installed but sqlite is not?

**Comment by coleifer** [comment]
I have no idea what conda is?

**Comment by happylrd** [comment]
Anaconda is a python and R distribution. It aims to provide everything you need (python wise) for data science "out of the box".
It includes:
- The core python language
- 100+ python "packages" (libraries)
- Spyder (IDE/editor - like pycharm) and Jupyter
- **conda, Anaconda's own package manager**, used for updating Anaconda and packages

See [here](https://stackoverflow.com/questions/49471840/anaconda-error-cannot-open-include-file-sqlite3-h-no-such-file-or-directory), it may help to update the version of peewee.

**Comment by jwkvam** [comment]
@happylrd probably better to ask [here](https://github.com/conda-forge/peewee-feedstock) since it seems the conda package wasn't made by coleifer.

It looks like @pmlandwehr and @dharhas have marked themselves as the maintainers of that.

**Comment by coleifer** [comment]
Let me rephrase: I totally don't care what conda is.

I'd suggest you check it out with whoever the maintainers are.

**Comment by coleifer** [comment]
For all you winblows users out there, I've added an option to the setup.py script to skip building SQLite-specific C extensions. Docs here:

http://docs.peewee-orm.com/en/latest/peewee/installation.html#skip-compilation-of-sqlite-extensions

**Comment by coleifer** [comment]
This environment var is supported in the latest version on PyPI -- 3.1.7, which I just pushed. Hope this helps!
